application/comit_node/src/bam_api/rfc003/swap.rs

```diff
@@ -14,6 +14,7 @@ use bam::{
     Status,
 };
 use futures::future::{self, Future};
+use log::{error, warn};
```

Comment:
❌ While I understand the risk of "poisoning the global space" I really think that for log macros, having to import them individually in each files is a pain.

Reply:
I'd say we could decide this in #487. For now just let them be global. If we are able to design things so that things are only logged at key places then we wouldn't have to import them in many places. i.e. You should be forced to think about actually logging things in random places.

Reply:
> While I understand the risk of "poisoning the global space" I really think that for log macros, having to import them individually in each files is a pain.

I understand your point and would like to provide some data that might shed a different light onto this situation:

Here are some numbers for the comit_node crate:

- `use log::`: 22
- `use serde::`: 33
- `use std::`: 63
- `use futures::`: 18

I think, it is just a question of what we are used to. Log macros where "just" available. That is why it seems like a pain to import them now. Overall though, I think they are just symbols like any other. Also the numbers show that there is symbols that we have to import today so I don't think they deserve a special treatment.

Reply:
@D4nte @LLFourn @luckysori 

We could also refer to those macros through an absolute path: `log::warn!("I am a warning!")`.

Thoughts?

Reply:
> We could also refer to those macros through an absolute path:
> `log::warn!("I am a warning!")`.
> 
> Thoughts?

I like that.